esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -456,6 +545,9 @@ void EsdlServiceImpl::configureTargets(IPropertyTree *cfg, const char *service)
 
         MapStringToMyClass<IEspPlugin> localCppPluginMap;
 
+        StringBuffer serviceSecurity(definition_cfg->queryProp("@auth_feature"));
+        m_bindingAccessMaps.kill();
```

Comment:
Just checking, but you made sure this doesn't leak the contents?

Reply:
I've run valgrind. Unless I'm completely misreading the output, no leaks are reported with configureTargets in the allocation stack trace. Is there something else I should be looking for?